torchgeo/datasets/bigearthnet.py

```diff
@@ -573,3 +585,441 @@ def plot(
         if suptitle is not None:
             plt.suptitle(suptitle)
         return fig
+
+
+class BigEarthNetV2(NonGeoDataset):
```

Comment:
Could also subclass BigEarthNet if you want less code duplication. Or make a shared base class. Depends on how similar they are.